tt_metal/host_api.hpp

```diff
@@ -127,6 +129,9 @@ KernelHandle CreateKernel(
     const std::variant<CoreCoord, CoreRange, CoreRangeSet> &core_spec,
     const std::variant<DataMovementConfig, ComputeConfig, EthernetConfig> &config);
 
+bool FindPersistentKernel(const std::string &file_name, const std::variant<CoreCoord, CoreRange, CoreRangeSet> &core_spec, const std::variant<DataMovementConfig, ComputeConfig, EthernetConfig> &config);
```

Comment:
Feedback to a method name
"Find" to me should return what you where looking for.

This looks more like a check if core spec contains the kernel. 
maybe something like `IsPersistentKernelAvailable`?